docs/rancher/node/rke2-cluster.md

```diff
@@ -43,9 +42,24 @@ Users can create a RKE2 Kubernetes cluster from the **Cluster Management** page
 8. Enter **Image** (required).
 9. Enter **Network Name** (required).
 10. Enter **SSH User** (required).
-11. Click **Create**.
+11. Configure the **Show Advanced > User Data** to enable the VMs to install required packages.
+```yaml
+#cloud-config
+packages:
+  - iptables
+```
+
+:::note
+
+Calico and Canal requires the `iptables` or `xtables-nft` package to be installed on the node.
```

Comment:
```suggestion
Calico and Canal require the `iptables` or `xtables-nft` package to be installed on the node, for more details, please refer to the [RKE2 known issues](https://docs.rke2.io/known_issues/#canal-and-ip-exhaustion).
```

Reply:
Also, what about the RKE1 cluster, will it also be required for newer versions?

Reply:
RKE1 cluster install docker by `https://releases.rancher.com/install-docker/20.10.sh`. iptables is in docker-ce‘s dependence
```bash
root@rke1:/home/ubuntu# iptables
iptables v1.8.4 (legacy): no command specified
Try `iptables -h' or 'iptables --help' for more information.
```
```bash
root@rke1:/home/ubuntu# apt show docker-ce
Package: docker-ce
Version: 5:20.10.21~3-0~ubuntu-focal
Priority: optional
Section: admin
Maintainer: Docker <support@docker.com>
Installed-Size: 88.5 MB
Depends: containerd.io (>= 1.4.1), docker-ce-cli, iptables, libseccomp2 (>= 2.3.0), libc6 (>= 2.8), libdevmapper1.02.1 (>= 2:1.02.97), libsystemd0
Recommends: apparmor, ca-certificates, docker-ce-rootless-extras, git, libltdl7, pigz, procps, xz-utils
Suggests: aufs-tools, cgroupfs-mount | cgroup-lite
Conflicts: docker (<< 1.5~), docker-engine, docker-engine-cs, docker.io, lxc-docker, lxc-docker-virtual-package
Replaces: docker-engine
Homepage: https://www.docker.com
Download-Size: 20.5 MB
APT-Sources: https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
Description: Docker: the open-source application container engine
 Docker is a product for you to build, ship and run any application as a
 lightweight container
 .
 Docker containers are both hardware-agnostic and platform-agnostic. This means
 they can run anywhere, from your laptop to the largest cloud compute instance and
 everything in between - and they don't require you to use a particular
 language, framework or packaging system. That makes them great building blocks
 for deploying and scaling web apps, databases, and backend services without
 depending on a particular stack or provider.

N: There are 28 additional records. Please use the '-a' switch to see them.
```